.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    # trying and staging branches are for BORS config
+    branches:
+      - trying
+      - staging
+      - main
+
+jobs:
+  playground_build_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10', '12', '14' ]
+    name: playground-build (Node.js ${{ matrix.node }})
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Build Strapi playground
+        run: yarn playground:build
+
+  linter_check:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10', '12', '14' ]
```

Comment:
No need to run the linter in 3 different versions. The goal is to have a unified code base, so no need to use a matrix for this :)

Reply:
I have added these tests to ensure our dev environment work on all node environments. I'm not sure I understand what you mean with an unified code base?

Reply:
Yes, the environment, so the test. No the linter

Reply:
Ah I see! What if the linter has been updated to a version not compatible with node 10 for example? The same way `jest` which does the integrations tests stopped working on node 10 in meilisearch-js and instant-meilisearch?